laws/src/main/scala/cats/laws/NonEmptyTraverseLaws.scala

```diff
@@ -1,6 +1,8 @@
 package cats.laws
 
-import cats.{Apply, Id, NonEmptyTraverse, Semigroup}
+// The `catsInstancesForId` import is necessary to work around a Dotty
+// issue related to https://github.com/lampepfl/dotty/issues/9067.
```

Comment:
The issue seems resolved in 0.25.0's release candidates. I guess we like to stay on stable versions ? I'm just curious.

Reply:
@barambani I thought about using 0.25.0-RC2, and I don't think we should avoid the Dotty RCs on principle, but the 3.2.0 ScalaTest dependencies are currently only published for 0.24.0, and I think it's more important to make it easy for people to work on the test issues.

Reply:
In general, I think the *right* thing for us (and the whole ecosystem) to do would be to publish for the latest stable Dotty (in this case, 0.24.0) *as well as* the exact latest RC (in this case, 0.25.0-RC2). Obviously it's a pain keeping up with RCs, but it helps quite a bit. Dependencies which don't do this, particularly those which lack macro usage, effectively force their downstream dependents to fall back on the 2.13 cross-publish, which is annoying.